test/storage/projected_row_test.cpp

```diff
@@ -131,7 +133,7 @@ TEST_F(ProjectedRowTests, Alignment) {
     storage::BlockLayout layout = StorageTestUtil::RandomLayout(common::Constants::MAX_COL, &generator_);
 
     // generate a random projectedRow
-    std::vector<uint16_t> all_col_ids = StorageTestUtil::ProjectionListAllColumns(layout);
+    std::vector<col_id_t> all_col_ids = StorageTestUtil::ProjectionListAllColumns(layout);
```

Comment:
Do we want to replace these with `auto` while we're in here? The function name seems descriptive enough to imply what's going on.

Reply:
as discussed we can make things explicit when it's not too much trouble, and when tidy is not yelling.